chapters/publication.tex

```diff
@@ -99,91 +174,259 @@ \section{Collaborating on academic writing}
+you can translate the raw document into Word
+(or a number of other formats)
+by running the following code from the command line:
 
 \codeexample{pandoc.sh}{./code/pandoc.sh}
 
+\noindent The last portion after \texttt{csl=} specifies the bibliography style.
+You can download a CSL (Citation Styles Library) file\sidenote{
+  \url{https://github.com/citation-style-language/styles}}
+for nearly any journal and have it applied automatically in this process.
 Therefore, even in the case where you are requested to provide
 \texttt{.docx} versions of materials to others, or tracked-changes versions,
 you can create them effortlessly,
-and use Word's compare feature to generate a single integrated tracked version.
+and use external tools like Word's compare feature
+to generate integrated tracked versions when needed.
+
+\subsection{Getting started with \LaTeX\ via Overleaf}
+
+\LaTeX\ is a challenging tool to get started using,
+but the control it offers over the writing process is invaluable.
+In order to make it as easy as possible for your team
+to use \LaTeX\ without all members having to invest in new skills,
+we suggest using the web-based Overleaf implementation as your first foray into \LaTeX\ writing.\sidenote{
+  \url{https://www.overleaf.com}}
+While the Overleaf site has a subscription feature that offers some useful extensions,
+its free-to-use version offers basic tools that are sufficient
+for a broad variety of basic applications,
+up to and including writing a complete academic paper with coauthors.
+
+Overleaf's implementation of \LaTeX\ is suggested here for several reasons.
+Since it is completely hosted online,
+it avoids the inevitable troubleshooting of setting up a \LaTeX\ installation
+on various personal computers run by the different members of your team.
+It also automatically maintains a single master copy of the document
+so that different writers do not create conflicted or out-of-sync copies,
+and allows inviting collaborators to edit in a fashion similar to Google Docs.
+Overleaf also offers a basic version history tool that avoids having to use separate software.
+Most importantly, it provides a ``rich text'' editor
+that behaves pretty similarly to familiar tools like Word,
+so that people can write into the document without worrying too much
+about the underlying \LaTeX\ coding.
+Overleaf also offers a convenient selection of templates
+so it is easy to start up a project and see results right away.
+
+On the downside, there is a small amount of up-front learning required,
+continous access to the Internet is necessary,
+and updating figures and tables requires a bulk file upload that is tough to automate.
+One of the most common issues you will face using Overleaf's ``rich text'' 
+editor will be special characters
+which, because of code functions, need to be handled differently than in Word.
+Most critically, the ampersand (\texttt{\&}), percent (\texttt{\%}), and underscore (\texttt{\_})
```

Comment:
These examples are note specific to Overleaf. They should probably go in the section that talks about LaTeX in general. Similarly to BibTex